### PR TITLE
Update package.json file

### PR DESCRIPTION
**Pull request by alejandronanez**
- Add engines field to sugges npm 5.3.0 or greater
- Add missing key fields in the package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "engines": {
+    "npm": ">=5.3"
```
Comment (andrewda) [inline package.json]: why do we need to specify an npm version? should we also specify a node version?

Comment (alejandronanez) [reply package.json]: You can specify both actually. You can have npm installed on different node versions, that’s why I added only npm.

Comment (andrewda) [reply package.json]: alright sounds good

File: package.json [hunk 2]
```diff
@@ -7,5 +10,10 @@
   },
   "devDependencies": {
     "sw-precache": "^5.1.1"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gitpoint/gitpoint-site"
```
Comment (machour) [inline package.json]: It's git-point-site (and why not point to the git-point-site point repo instead?)

Comment (alejandronanez) [reply package.json]: I missed that one! Updated in #18

**Pull request merged**